src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Tests
+{
+    // TODO: #13187
+    [PlatformSpecific(TestPlatforms.Windows)]
+    public class HttpAuthenticationTest
+    {
+        private const string Basic = "Basic";
+        private const string TestUser = "testuser";
+        private const string TestPassword = "testpassword";
+
+        [Theory]
+        [InlineData(AuthenticationSchemes.Basic)]
+        [InlineData(AuthenticationSchemes.Basic | AuthenticationSchemes.None)]
+        [InlineData(AuthenticationSchemes.Basic | AuthenticationSchemes.Anonymous)]
+        public async Task TestBasicAuthentication(AuthenticationSchemes authScheme)
```

Comment:
Nit: most System.Net tests follow a particular naming convention. Since this is a new contract and test I don't care that much but the difference will become more apparent as we port NetFX tests.